packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/__snapshots__/ChainHistoryBuilder.test.ts.snap

```diff
@@ -99,7 +99,7 @@ Map {
 
 exports[`ChainHistoryBuilder queryProtocolParams query protocol parameters from last epoch 1`] = `
 Object {
-  "coinsPerUtxoWord": NaN,
+  "coinsPerUtxoByte": NaN,
```

Comment:
Is `NaN` ok here? Maybe we should include a Babbage block to test that this param exists.

I'm wondering why it was `NaN` before too, maybe this is an actual bug :thinking: